ONSdigital/dp-developer-site
# Added example responses and updated styling to endpoint pages

### What

- Updated the styling for the responses pages to reflect that which is on the prototype.
- Added `ExampleResponse` property to `MethodResponse` struct and `json` marshal code to render example responses onto the endpoint pages.

### How to review

- Pull changes and visit an endpoint page, e.g., http://localhost:23600/filter/filter-outputs-filter_output_id/
- Check that styling aligns with that of the prototype
- Check example response rendering as expected

### Who can review

Anyone but me.
